llm/README.md

```diff
@@ -0,0 +1,71 @@
+# Personal CHTC Submit Template for LLM Fine-Tuning
+
+Use Case: Fine-tuning large language models on CHTC and track it with `Weights & Biases`.
+
+![WANDB](wandb.png)
+
+## Quick start
+
+1. Store your WANDB credentials and `STAGING_DIR` path in an environment file named .env. See the provided [example](.env.example).
+1. Update the `run_name` in `run.sh`. This will be utilized as the WANDB tracking ID, and checkpoints will be saved in `STAGING_DIR/run_name/`.
```

Comment:
I believe we can move this to the submit file.

Reply:
My suggested changes now move the run name to the submit file so the instructions can be modified to update that file with the run name and any other necessary changes. The script only needs to be modified to disable W&B logging.

Reply:
Done in Christina's review suggestions.